orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -57,17 +57,15 @@ def __init__(self):
         self.doc_webview = None         # WebView for showing content
         self.search_features = []       # two copies are needed since Display allows drag & drop
         self.display_list_indices = [0]
+        self.matches = 0                # Matches of the query
 
         # Info attributes
         self.update_info()
         info_box = gui.widgetBox(self.controlArea, 'Info')
-        gui.label(info_box, self, 'Documents: %(n_documents)s')
-        gui.label(info_box, self, 'Preprocessed: %(is_preprocessed)s')
-        gui.label(info_box, self, '  ◦ Tokens: %(n_tokens)s')
-        gui.label(info_box, self, '  ◦ Types: %(n_types)s')
-        gui.label(info_box, self, 'POS tagged: %(is_pos_tagged)s')
-        gui.label(info_box, self, 'N-grams range: %(ngram_range)s')
+        gui.label(info_box, self, 'Tokens: %(n_tokens)s')
+        gui.label(info_box, self, 'Types: %(n_types)s')
         gui.label(info_box, self, 'Matching: %(n_matching)s')
+        gui.label(info_box, self, 'Matched queries: %(n_words)s')
```

Comment:
"Matched queries" is bad, but I cannot think of a better label. Ideas?

Reply:
Currently, I cannot get any better idea. My suggestion is maybe `matching objects`.

Reply:
But I don't like it either